FIWARE-adapter/config/APIClientConfiguration.json

```diff
@@ -0,0 +1,14 @@
+{
+    "ip": "5.53.108.182",
+    "port": "8668",
+    "fiware_service": "carouge",
+    "entity_id": "urn:ngsi-ld:FlowerBed:FlowerBed-1",
```

Comment:
Should we keep 2 separate configs? One for API endpoint, the other for concrete entity id? Should we make config so that it can connect to multiple sources?

Reply:
Perhaps not, because we are using separate calls. But what if there is some sort of authentication at some point?